iroh/Cargo.toml

```diff
@@ -59,12 +59,12 @@ reqwest = { version = "0.12", default-features = false, features = [
 ] }
 ring = "0.17"
 rustls = { version = "0.23", default-features = false, features = ["ring"] }
-serde = { version = "1", features = ["derive", "rc"] }
+serde = { version = "1.0.219", features = ["derive", "rc"] }
 smallvec = "1.11.1"
 strum = { version = "0.26", features = ["derive"] }
-stun-rs = "0.1.5"
+stun-rs = "0.1.11"
 thiserror = "2"
-tokio = { version = "1", features = [
+tokio = { version = "1.44.1", features = [
```

Comment:
That's pretty recent. On the plus side it allows using https://docs.rs/tokio/latest/tokio/sync/oneshot/struct.Receiver.html#method.is_terminated I think...